test/csuite/timestamp_abort/main.c

```diff
@@ -466,7 +466,7 @@ thread_run(void *arg)
              * Set the active timestamp to the first of the three timestamps we reserve for use this
              * iteration. Use the first reserved timestamp.
              */
-            active_ts = RESERVED_TIMESTAMPS_FOR_ITERATION(td->info, i);
+            active_ts = RESERVED_TIMESTAMPS_FOR_ITERATION(td->info, iter);
```

Comment:
This seems to be a bug, since ```i``` starts with a multiple of ```WT_BILLION```, out timestamps started at a billion.

Also, I am not sure if it is better to use a separate iter (my preference), or to replace it with ```i % WT_BILLION```. Also, the structure ```thread_data``` has variables ```start``` and ```info``` - we could likely eliminate ```start``` as it is derivable from ```info``` as ```WT_BILLION * info```. I reckon we could keep it as such to allow for a future use case of having different key spaces. Other similar patterned csuite examples have the same key spaces. Any thoughts?